pulsar-io/jdbc/core/pom.xml

```diff
@@ -58,6 +58,13 @@
       <groupId>com.fasterxml.jackson.dataformat</groupId>
       <artifactId>jackson-dataformat-yaml</artifactId>
     </dependency>
+
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java</artifactId>
+      <version>3.22.4</version>
```

Comment:
Let's use the version in `dependencyManagement` in parent pom.

Also, cloud we use `provided` scope? (N.B. I don't take a considerate thought).

Reply:
+1 we need to use `provided` to avoid runtime clashes
Also get the already declared version

Reply:
I set the version to `${protobuf3.version}` from `pulsar/pom.xml` and added the provided scope.
Is this what you ment by `dependencyManagement` ?

Reply:
@bpereto Just remove the `<version>` line. We already import:

```xml
      <dependency>
        <groupId>com.google.protobuf</groupId>
        <artifactId>protobuf-bom</artifactId>
        <version>${protobuf3.version}</version>
        <type>pom</type>
        <scope>import</scope>
      </dependency>
```

Reply:
@bpereto Also, please verify that `provided` statisfy your use case. One downside of `provided` is that - I'm not sure - if we don't pack protobuf into the NARball, it's possible to meet `NoClassDefFoundError`.

Reply:
I will test it with data, currently the sink is starting with the proposed changes.